webapp/src/main/java/org/exoplatform/officeonline/portlet/EditorPortlet.java

```diff
@@ -100,7 +100,9 @@ public void view(RenderRequest request, RenderResponse response) throws IOExcept
         if (validAction(node, action)) {
           String actionURL = wopiService.getActionUrl(requestInfo, fileId, null, action);
           String filename = node.getName();
-          callModule("officeonline.initEditor(" + token.toJSON() + ", \"" + actionURL + "\", \"" + filename + "\");");
+          String explorerUrl = wopiService.getExplorerURL(node, config.getBaseUrl());
+          String versionsUrl = explorerUrl + "&versions=true"; 
```

Comment:
StringBuilder is better from perf PoV

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/28/commits/09dd43c4c6497a4cd87cc727a522b7c45c3eefdd